controllers/apply/form-router/index.js

```diff
@@ -315,6 +315,16 @@ function initFormRouter({
                         'status'
                     );
 
+                    if (currentApplication.isExpired) {
```

Comment:
Mind adding this a subsequent middleware function?  Wary of this one getting a bit overloaded. Makes sense to do this early on though.

Reply:
Good call 👍 done